apps/studio/components/interfaces/Settings/API/JWTSettings.tsx

```diff
@@ -93,117 +138,161 @@ const JWTSettings = () => {
-                    <div className="w-full space-y-2">
-                      <div className="flex items-center justify-between w-full">
-                        <div className="flex flex-col space-y-1">
-                          <p className="text-sm">Generate a new JWT secret</p>
-                          <p className="text-sm opacity-50">
-                            A random secret will be created, or you can create your own.
-                          </p>
-                        </div>
-                        <div className="flex flex-col items-end">
-                          {isUpdatingJwtSecret ? (
-                            <Button loading type="secondary">
-                              Updating JWT secret...
-                            </Button>
-                          ) : !canGenerateNewJWTSecret ? (
-                            <ButtonTooltip
-                              disabled
-                              type="default"
-                              iconRight={<ChevronDown size={14} />}
-                              tooltip={{
-                                content: {
-                                  side: 'bottom',
-                                  text: 'You need additional permissions to generate a new JWT secret',
-                                },
-                              }}
-                            >
-                              Generate a new secret
-                            </ButtonTooltip>
-                          ) : (
-                            <DropdownMenu>
-                              <DropdownMenuTrigger asChild>
-                                <Button type="default" iconRight={<ChevronDown size={14} />}>
-                                  <span>Generate a new secret</span>
-                                </Button>
-                              </DropdownMenuTrigger>
-                              <DropdownMenuContent align="end" side="bottom">
-                                <DropdownMenuItem
-                                  className="space-x-2"
-                                  onClick={() => setIsGeneratingKey(true)}
-                                >
-                                  <RefreshCw size={16} />
-                                  <p>Generate a random secret</p>
-                                </DropdownMenuItem>
-                                <DropdownMenuSeparator />
-                                <DropdownMenuItem
-                                  className="space-x-2"
-                                  onClick={() => setIsCreatingKey(true)}
-                                >
-                                  <PenTool size={16} />
-                                  <p>Create my own secret</p>
-                                </DropdownMenuItem>
-                              </DropdownMenuContent>
-                            </DropdownMenu>
-                          )}
-                        </div>
+      <Form
```

Comment:
okay for now but jfyi that `<Form>` is deprecated - use `<Form_Shadcn>` with react hook form instead